templates/_partials/user/account_dropdown.html.twig

```diff
@@ -8,6 +8,13 @@
         {% trans %}navbar.myAccount{% endtrans %}
       </a>
     </li>
+    {% if context_channel.channel.code == 'pro' %}
+      <li><a href="{{ path('pro_account_cart', {'redirect_to': app.request.uri, '_channel_code' : 'web'}) }}">{% trans %}navbar.lambdaCart{% endtrans %}</a></li>
```

Comment:
Using the `_channel_code` parameter only works in dev, because the [FakeChannelCodeProvider](https://github.com/Sylius/Sylius/blob/8e50539ce5015d98403b6c25637e4b6bb1b6ddf9/src/Sylius/Bundle/ChannelBundle/Context/FakeChannel/FakeChannelCodeProvider.php) is injected (see how the [dependency injection](https://github.com/Sylius/Sylius/blob/8e50539ce5015d98403b6c25637e4b6bb1b6ddf9/src/Sylius/Bundle/ChannelBundle/DependencyInjection/SyliusChannelExtension.php#L34-L38) works). This won't work in production.